### DISCUSSION TITLE
Mapping between different frameworks

### DISCUSSION THREAD

**Discussion by 42mst** [intuitem/ciso-assistant-community#239] [Ideas]
- Are there any plans to create cross mapping between the frameworks?
- This could be useful for projects where you try to find out for example how NIS2-ready you are.

**Comment by eric-intuitem** [comment]
Hi @42mst this is indeed high on our roadmap, please have a look to our specification in documentation/data-model.md about the mapping feature.

**Reply by 42mst** [reply]
@eric-intuitem 
Do you think this documentation/data-model.md model is sufficient enough in the long run?

Initial situation:
- ISO27001 <-> NIST
- ISO27001 <-> CCB
- Client A is currently NIST compliant 

Goal:
- Client A needs to implement the latest CCB requirements to be NIS2 compliant.

Current Model:
- Currently that means in your model that each control needs to be mapped against each other control which would create a huge overhead. 
- Maybe ( I´m not sure about that) it would make more sense to have a - let´s say - top level framework, like the ISO27001
- All controls need to be mapped to that
- To achieve the described goal you would have/need to implement a transitive relationship between the frameworks and by that the controls
-->  NIST <-> ISO27001 <-> CCB
- By that the top-level framework will always be the connecting element.
